COLLABORATOR_GUIDE.md

```diff
@@ -515,6 +515,12 @@ Run tests (`make -j4 test` or `vcbuild test`). Even though there was a
 successful continuous integration run, other changes may have landed on master
 since then, so running the tests one last time locally is a good practice.
 
+Validate that the commit message is properly formatted using [core-validate-commit](https://github.com/evanlucas/core-validate-commit).
+
+```text
+$ core-validate-commit
```

Comment:
`core-validate-commit` is also capable of validating multiple commits. Maybe

```
# Validate the HEAD commit
$ core-validate-commit

# Validate a single sha:
$ core-validate-commit 64c87e2

# Validate from a specific sha to HEAD:
$ git rev-list 287bdab..HEAD | xargs core-validate-commit
```

Reply:
This should be done after the rebase so they should only need to validate head, right?

Reply:
@bmeck If the person doing this is landing multiple commits, then they should validate all of them. Most of the times we are squashing all commits into one but there are times when a PR needs to be land as multiple commits (and it's easier to forget to add metadata to the commits between the two heads, I would say)

Reply:
updated